src/XummSdk.php

```diff
@@ -86,6 +77,22 @@ public function getKycStatusByUserToken(string $userToken): KycStatus & XummResp
         return $this->client->post(Request::getKycStatusByUserToken, $body);
     }
 
+    public function verifyUserTokens(string ...$userTokens): UserTokenValidityRecordList & XummResponse
```

Comment:
Works like a charm, no specific preference for spread operator vs direct Array passing. JS/TS SDK implements passing an array:
https://github.com/XRPL-Labs/XUMM-SDK#sdkverifyusertokensstring--sdkverifyusertokenstring

Did you use an Array destructed to strings as args for consistency / preference / ..? (curious)

Reply:
For typehinting! PHP doesn't support typehinting array items (yet) unfortunately, so this is usually how I enforce argument types

Reply:
Ahhh, learned something. Great. Thank you :D